mmcdata.py

```diff
@@ -141,8 +150,12 @@ def _process_data(self,data,convert_ft_to_m=False):
         else:
             # Otherwise expect heights in meters as they should be
             self.dataDict['z'] = np.asarray(z)
-        self.dataDict['u']     = np.asarray(u)
-        self.dataDict['v']     = np.asarray(v)
+        if map_to_met_coords:      #map TTU-somic (unorth, vwest) sonic coords to standard meteorology coordinates
```

Comment:
I would argue that case-specific data manipulations should be left to the user, e.g., as a processing step in `assessment`, rather than as a hard-coded operation within `mmctools`.

Reply:
This was the point of adding the kwarg. Now someone can explicitly include the flag in the data converter or not.  While the conversion can be done by the user explicitly now there is a permanent section of code in the tools that knows TTU sonic data isn't in traditional met coords and can do the conversion.  In some ways I see this as much as a form of permanent documentation regarding the TTU tower as anything else.